composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":             ">=5.3.2",
         "ext-curl":        "*",
-        "guzzle/guzzle":   "~3.7"
+        "guzzlehttp/guzzle": "~3.7 || ~4.0 || ~5.0 || ~6.0"
```

Comment:
guzzle 3 does not have the same package name.

Btw, this also means that it is perfectly possible to install guzzle 3 in a project already using Guzzle 5 or 6. This makes this PR less needed than in other projects using Guzzle 5